projects/demo/src/modules/components/push/examples/1/index.less

```diff
@@ -10,6 +10,14 @@
     max-inline-size: 100%;
     border-radius: var(--tui-radius-l);
     margin-bottom: 1rem;
+
```

Comment:
Please use the fix I suggested in the issue. The problem is not with gap per say, it's the background of the wrapper that's missing. I believe the gap is correct.

Reply:
A wrapper contains 2 push elements as child elements. If we use the suggested fix, it merges both push elements. Can you confirm is wrapper necessary for each push element?

Reply:
Yes, the idea of this example is to show a list of push elements as in a dropdown when clicking on Bell icon with notifications, hence first one having (X) icon as if it's there to close the whole list.

Reply:
Thanks for clarifying the requirements Alex, I have resolved the PR comment. Please checkout.